hack/overlays/overlay-image.yaml

```
@@ -0,0 +1,34 @@
+#@ load("@ytt:overlay", "overlay")
+#! rewrite image and image pull policy
+
+#@ ko_import_path = "ko://github.com/projectcontour/contour/cmd/contour"
+
+#@overlay/match by=overlay.all,expects=17
+#@overlay/match-child-defaults when=1
+---
+spec:
+  template:
+    spec:
+      containers:
+        #@overlay/match by="name"
+        - name: contour
+          #@overlay/replace via=lambda left, right: left.replace(left, ko_import_path)
+          image:
+          imagePullPolicy: IfNotPresent
+        #@overlay/match by="name"
+        - name: shutdown-manager
+          #@overlay/replace via=lambda left, right: left.replace(left, ko_import_path)
+          image:
```

Is it possible to just have 

```suggestion
          image: ko://github.com/projectcontour/contour/cmd/contour
```

like you do lower in the file?

---

Yes 😂 , I don't know why I was using lamba for this...